packages/mobile/src/screens/profile-screen/ProfileHeader/SocialLink.tsx

```diff
@@ -18,6 +20,9 @@ import { EventNames } from 'app/types/analytics'
 
 import { useSelectProfile } from '../selectors'
 
+const prependProtocol = (url: Nullable<string>) =>
```

Comment:
Seems like it belongs in a more common utils location and I'm surprised we don't already have a util that does this?

Reply:
yeah let me see if there is a utility